R/middleware.R

```diff
@@ -383,6 +385,30 @@ HandlerManager <- R6Class("HandlerManager",
   )
 )
 
+getResponseContentLength <- function(response, deleteOwnedContent) {
+  force(deleteOwnedContent)
+
+  result <- if (is.character(response$content)) {
```

Comment:
It looks like you're trying to be strict about checking the input, and so it might be good to check that the length is 1.